src/visualization/interaction/OrbitControls.js

```diff
@@ -61,17 +69,19 @@ ROS3D.OrbitControls = function(options) {
   };
   var state = STATE.NONE;
 
-  // add the axes for the main coordinate frame
   this.axes = new ROS3D.Axes({
```

Comment:
`this.axes` isn't used if `displayPanAndZoomFrame` is `false`. Could you move this under if statement? And update `showAxes()` something like
```
 if(this.displayPanAndZoomFrame){
 var that = this;                                                                                                                                                                                                                                                           
  this.axes.traverse(function(obj) {                                                                                                   
    obj.visible = true;                                                                                                                
  });                                                                                                                                  
  if (this.hideTimeout) {                                                                                                              
    clearTimeout(this.hideTimeout);                                                                                                    
  }                                                                                                                                    
  this.hideTimeout = setTimeout(function() {                                                                                           
    that.axes.traverse(function(obj) {                                                                                                 
      obj.visible = false;                                                                                                             
    });                                                                                                                                
    that.hideTimeout = false;                                                                                                          
  }, 1000);                                                                                                                            
}
```

Reply:
And [line490-492](https://github.com/RobotWebTools/ros3djs/blob/develop/src/visualization/interaction/OrbitControls.js#L490)